api/resources/views/vendor/mail/html/promotion/button.blade.php

```diff
@@ -0,0 +1,13 @@
+<table width="100%" border="0" cellpadding="0" cellspacing="0">
```

Comment:
Do we need to export all those blade files? If there is an update, we'll need to update all manually.
Nothing is changed I guess to this file